Dockerfile

```diff
@@ -181,7 +181,6 @@ COPY hack/dockerfile/install/$INSTALL_BINARY_NAME.installer ./
 RUN PREFIX=/opt/$INSTALL_BINARY_NAME ./install.sh $INSTALL_BINARY_NAME
 
 
-
```

Comment:
nit: unrelated change

Reply:
It's not unrelated, I originally did the work here similar to runc/tini etc., however, I realized that the binaries would end up in the wrong location and should really only be built as part of the cross compile. It seems I removed an extra blank line by mistake when deleting my changes.

Reply:
If you can revert to keep the diff focussed on the actual changes, that'd be appreciated 🤗